mkdocs.yml

```diff
@@ -127,10 +127,10 @@ nav:
       - Create a Backstage Plugin: 'plugins/create-a-plugin.md'
       - Plugin Development: 'plugins/plugin-development.md'
       - Structure of a plugin: 'plugins/structure-of-a-plugin.md'
-      - Plugin Development: 'plugins/plugin-development.md'
-      - Integrate into the Software Catalog: 'plugins/integrating-plugin-into-software-catalog.md'
+      - Integrate into the Software Cagittalog: 'plugins/integrating-plugin-into-software-catalog.md'
```

Comment:
Small typo here

Reply:
Ha you can see I typed git and cursor was in the wrong place and it edited the file 🤦‍♂️ good catch